htdocs/expedition/class/expedition.class.php

```diff
@@ -648,6 +649,17 @@ public function create($user, $notrigger = 0)
 					}
 				}
 
+				// update ref
+				$initialref = '(PROV'.$this->id.')';
+				if (!empty($this->ref)) {
+					$initialref = $this->ref;
+				}
+
+				$sql = 'UPDATE '.MAIN_DB_PREFIX.$this->table_element." SET ref='".$this->db->escape($initialref)."' WHERE rowid=".((int) $this->id);
```

Comment:
Why not putting this (using $this->ref instead of (PROVx) directly at line 497 (to avoid 2 updates ?